src/library/FOSSBilling/Config.php

```diff
@@ -106,7 +106,7 @@ public static function setConfig(array $newConfig, bool $clearCache = true): voi
 
             try {
                 $filesystem->remove(PATH_CACHE);
-                $filesystem->mkdir(PATH_CACHE, 0755);
+                $filesystem->mkdir(PATH_CACHE, 0o755);
```

Comment:
This change was done by PHP-CS-Fix and is from a rule that's part of their PHP 8.1 migration